### DISCUSSION TITLE
Elements from Run dropdown in global menu should provide feedback

### DISCUSSION THREAD

**Discussion by pedrorrodriguez** [aurelienpierreeng/ansel#789] [Ideas]
At the moment you don't know if it's doing anything or if it's finished doing whatever action you chose.

**Comment by aurelienpierre** [comment]
Preloading images has a progress bar. As for the others, they are almost immediate.

**Comment by pedrorrodriguez** [comment]
I figured, but since there's no feedback i didn't know. Perhaps a "Done!" popup would be better, for those almost instantaneous actions.

**Comment by Jiyone** [comment]
That's a good idea.
